azerothcore/azerothcore-wotlk
# feat(Core/Loot): Increase the max loot possible by 2

Before, it was 16, now let's use the max potential of the client.





## CHANGES PROPOSED:
-  Increase the max loot possible by 2 (from 16 to 18)

The idea came from a guy who researched it on TC https://github.com/TrinityCore/TrinityCore/issues/23948


## ISSUES ADDRESSED:

This is useful on certain bosses that drop a ton of items, some bosses drop 15 items by default, so if you increase the loot amount (with the configuration options) or add 2 new items, it will not show up and you're gonna debug this for days. At least, 2 more items can't hurt and it will only happen for custom servers


## TESTS PERFORMED:
- Tested ingame, works fine!
-



## HOW TO TEST THE CHANGES:

- Take a boss that drops a lot of items, like Azuregos, check his loot_creature_template and increase the amount of dropped item example mincount and maxcount to 10, then check ingame how many items it drops. Before PR, 16 max, after PR 18 max


## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
